integrations/ultralytics-yolov5/tutorials/sparsify-from-scratch.md

```diff
@@ -0,0 +1,409 @@
+      - model.6.m.0.cv2.conv.weight
+      - model.7.conv.weight
+      - model.8.m.0.cv2.conv.weight 
+    init_sparsity: eval(init_sparsity)
+    final_sparsity: 0.7602  
+    start_epoch: eval(pruning_start_epoch)
+    end_epoch: eval(pruning_end_epoch)
+    update_frequency: eval(pruning_update_frequency)
+   
+  - !GMPruningModifier
+    params:
+      - model.23.m.0.cv2.conv.weight
+    init_sparsity: eval(init_sparsity)
+    final_sparsity: 0.8453
+    start_epoch: eval(pruning_start_epoch)
+    end_epoch: eval(pruning_end_epoch)
+    update_frequency: eval(pruning_update_frequency) 
+
+  - !GMPruningModifier
+    params:
+      - model.21.conv.weight 
+    init_sparsity: eval(init_sparsity)
+    final_sparsity: 0.9002  
+    start_epoch: eval(pruning_start_epoch)
+    end_epoch: eval(pruning_end_epoch)
+    update_frequency: eval(pruning_update_frequency)
+
+knowledge_disitillation_modifiers:
+  - !PerLayerDistillationModifier
+    start_epoch: 0.0
+    end_epoch: eval(num_epochs - quantization_epochs)
+    gain: eval(per_layer_distillation_gain)
+    project_features: true
+    student_layer_names:
+    - model.0
+    - model.1
+    - model.2.cv1
+    - model.2.cv2
+    - model.2.cv3
+    - model.2.m.0.cv1
+    - model.2.m.0.cv2
+    - model.3
+    - model.4.cv1
+    - model.4.cv2
+    - model.4.cv3
+    - model.4.m.0.cv1
+    - model.4.m.0.cv2
+    - model.5
+    - model.6.cv1
+    - model.6.cv2
+    - model.6.cv3
+    - model.6.m.0.cv1
+    - model.6.m.0.cv2
+    - model.7
+    - model.8.cv1
+    - model.8.cv2
+    - model.8.cv3
+    - model.8.m.0.cv1
+    - model.8.m.0.cv2
+    - model.9.cv1
+    - model.9.cv2
+    - model.10
+    - model.13.cv1
+    - model.13.cv2
+    - model.13.cv3
+    - model.13.m.0.cv1
+    - model.13.m.0.cv2
+    - model.14
+    - model.17.cv1
+    - model.17.cv2
+    - model.17.cv3
+    - model.17.m.0.cv1
+    - model.17.m.0.cv2
+    - model.18
+    - model.20.cv1
+    - model.20.cv2
+    - model.20.cv3
+    - model.20.m.0.cv1
+    - model.20.m.0.cv2
+    - model.21
+    - model.23.cv1
+    - model.23.cv2
+    - model.23.cv3
+    - model.23.m.0.cv1
+    - model.23.m.0.cv2
+    - model.24.m.0
+    - model.24.m.1
+    - model.24.m.2
+
+quantization_modifiers:
+  - !QuantizationModifier
+    start_epoch: eval(num_epochs - quantization_epochs)
+    submodules:
+      - model
+    custom_quantizable_module_types: ['SiLU']
+    exclude_module_types: ['SiLU']
+    quantize_conv_activations: False
+    disable_quantization_observer_epoch: eval(num_epochs - quantization_epochs + 2)
+    freeze_bn_stats_epoch: eval(num_epochs - quantization_epochs + 1)
+```
+   
+There is is a lot here, but the important items are the `pruning_modifiers`, `distillation_modifiers`, and
+`quantization_modifiers`.
+
+The `pruning_modifiers` instruct SparseML to apply the Gradual Magnitude Pruning algorithm to various layers of
+the network. As you can see, the recipe specifies a target level of sparsity for each layer of the network.
```

Comment:
As you can see, the recipe specifies a target sparsity level for each network layer